src/index.js

```diff
@@ -402,14 +402,15 @@ var patchElement = function(
 }
 
 var createVNode = function(name, props, children, element, key, type) {
-  return {
+  var pureVNode = Object.create(null)
+  return Object.assign(pureVNode, {
```

Comment:
@cbbfcd Why are you using `Object.assign`?

Reply:
I realized this, so I can add a simple polyfill or use another method.